frontend/src/metabase/components/Schedule/components.tsx

```diff
@@ -174,7 +174,6 @@ export const AutoWidthSelect = (props: SelectProps) => {
         wrapper: {
           paddingRight: 0,
           marginTop: 0,
-          "&:not(:only-child)": { marginTop: "0" },
```

Comment:
Good find - I had added this to override the hack

Reply:
I added another commit that overrides the new label override to remove the margin again: d75cbc25780ad36c323b7e918252c10927871bf3.